tests/test_crf.py

```diff
@@ -470,9 +470,194 @@ def test_first_timestep_mask_is_not_all_on(self):
 
 if Version(torch.__version__) >= Version("1.9.0"):
 
-    def assert_close(actual, expected):
-        torch.testing.assert_close(actual, expected, atol=1e-12, rtol=1e-6)
+    def assert_close(actual, expected, atol=1e-12, rtol=1e-6):
+        torch.testing.assert_close(actual, expected, atol=atol, rtol=rtol)
 else:
 
-    def assert_close(actual, expected):
-        torch.testing.assert_allclose(actual, expected, atol=1e-12, rtol=1e-6)
+    def assert_close(actual, expected, atol=1e-12, rtol=1e-6):
+        torch.testing.assert_allclose(actual, expected, atol=atol, rtol=rtol)
```

Comment:
Please pass both `atol` and `rtol` as positional arguments if possible. Otherwise, feel free to ignore this comment.

Reply:
This change will not be necessary as I reverted the test changes.